libs/category/src/filters/behaviors/apply/apply-filter.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('daffApplyFilter', () => {
+  it('should apply a toggle request', () => {
```

Comment:
I don't understand what this function has to do with a toggle request. Should this be Equal vs Range instead of toggle vs idempotent?

Reply:
I don't think this spec name is correct. Damien and I did not do anything at all to the specs after a significant amount of work on the source files so most of them are likely wrong.